common/flatpak-dir.c

```diff
@@ -2356,25 +2365,28 @@ flatpak_dir_run_triggers (FlatpakDir   *self,
           g_debug ("running trigger %s at %s", name, basedir);
 
           argv_array = g_ptr_array_new_with_free_func (g_free);
-#ifdef DISABLE_SANDBOXED_TRIGGERS
-          g_ptr_array_add (argv_array, g_file_get_path (child));
-          g_ptr_array_add (argv_array, g_strdup (basedir));
-#else
-          g_ptr_array_add (argv_array, g_strdup (flatpak_get_bwrap ()));
-          g_ptr_array_add (argv_array, g_strdup ("--unshare-ipc"));
-          g_ptr_array_add (argv_array, g_strdup ("--unshare-net"));
-          g_ptr_array_add (argv_array, g_strdup ("--unshare-pid"));
-          g_ptr_array_add (argv_array, g_strdup ("--ro-bind"));
-          g_ptr_array_add (argv_array, g_strdup ("/"));
-          g_ptr_array_add (argv_array, g_strdup ("/"));
-          g_ptr_array_add (argv_array, g_strdup ("--proc"));
-          g_ptr_array_add (argv_array, g_strdup ("/proc"));
-          g_ptr_array_add (argv_array, g_strdup ("--dev"));
-          g_ptr_array_add (argv_array, g_strdup ("/dev"));
-          g_ptr_array_add (argv_array, g_strdup ("--bind"));
-          g_ptr_array_add (argv_array, g_strdup (basedir));
-          g_ptr_array_add (argv_array, g_strdup (basedir));
-#endif
+          if (disable_sandboxed_triggers)
+            {
+              g_ptr_array_add (argv_array, g_file_get_path (child));
+              g_ptr_array_add (argv_array, g_strdup (basedir));
```

Comment:
I feel like this branch is not needed (and wasn't before in the `#ifdef` case). The `g_file_get_path (child)` and `g_strdup (basedir)` are repeated after the conditional. I think the way you have this, it would run `trigger path trigger path`. That said, all the triggers ignore any arguments after the 1st, so it probably doesn't matter in practice.